diff --git a/cjs/apollo.js b/cjs/apollo.js
index c51c0b472753be0abee08107d8326a17ad25c5ae..b92a4929195ddfc7f6fcd1d09b93cc31e0c19139 100644
--- a/cjs/apollo.js
+++ b/cjs/apollo.js
@@ -2,7 +2,7 @@
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.hiveApollo = exports.createSupergraphManager = exports.createSupergraphSDLFetcher = void 0;
 const tslib_1 = require("tslib");
-const crypto_1 = require("crypto");
+const crypto_1 = require("node:crypto");
 const axios_1 = tslib_1.__importDefault(require("axios"));
 const client_js_1 = require("./client.js");
 const utils_js_1 = require("./internal/utils.js");
diff --git a/cjs/gateways.js b/cjs/gateways.js
index cad5a3ad5694fea08c700f1b70fde0638643490b..42965b28d837d4c93ebaf638b1238c22c807301f 100644
--- a/cjs/gateways.js
+++ b/cjs/gateways.js
@@ -2,7 +2,7 @@
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.createServicesFetcher = exports.createSchemaFetcher = void 0;
 const tslib_1 = require("tslib");
-const crypto_1 = require("crypto");
+const crypto_1 = require("node:crypto");
 const axios_1 = tslib_1.__importDefault(require("axios"));
 const version_js_1 = require("./version.js");
 function createFetcher({ endpoint, key }) {
diff --git a/cjs/internal/utils.js b/cjs/internal/utils.js
index 915986cc687bdf20b186e0e1f6012e24c069d503..444acde0b6aa0a3b4ae5470e23594cbe960a9bf7 100644
--- a/cjs/internal/utils.js
+++ b/cjs/internal/utils.js
@@ -1,7 +1,7 @@
 "use strict";
 Object.defineProperty(exports, "__esModule", { value: true });
 exports.logIf = exports.isHiveClient = exports.addProperty = exports.measureDuration = exports.cacheDocumentKey = exports.cache = exports.memo = exports.isAsyncIterable = exports.isAsyncIterableIterator = void 0;
-const crypto_1 = require("crypto");
+const crypto_1 = require("node:crypto");
 function isAsyncIterableIterator(value) {
     return typeof (value === null || value === void 0 ? void 0 : value[Symbol.asyncIterator]) === 'function';
 }
diff --git a/esm/apollo.js b/esm/apollo.js
index 092b52cfe514901cfcca353b710e7ea2a7abbcfb..30b0204837696b6d6ae09ddef6ca46005e71b0b9 100644
--- a/esm/apollo.js
+++ b/esm/apollo.js
@@ -1,4 +1,4 @@
-import { createHash } from 'crypto';
+import { createHash } from 'node:crypto';
 import axios from 'axios';
 import { createHive } from './client.js';
 import { isHiveClient } from './internal/utils.js';
diff --git a/esm/gateways.js b/esm/gateways.js
index 3ccaffea38f1847ac92a0cba47e23df77d543e49..4583c89150b1da3bf092827bfbc59c711f35a776 100644
--- a/esm/gateways.js
+++ b/esm/gateways.js
@@ -1,4 +1,4 @@
-import { createHash } from 'crypto';
+import { createHash } from 'node:crypto';
 import axios from 'axios';
 import { version } from './version.js';
 function createFetcher({ endpoint, key }) {
diff --git a/esm/internal/utils.js b/esm/internal/utils.js
index f1d575190d418791ba35f685f3f4ef7ca8fa1ea3..7e58ec2183b3f6e15b25aa80e8a67a6fa4bc1d14 100644
--- a/esm/internal/utils.js
+++ b/esm/internal/utils.js
@@ -1,4 +1,4 @@
-import { createHash } from 'crypto';
+import { createHash } from 'node:crypto';
 export function isAsyncIterableIterator(value) {
     return typeof (value === null || value === void 0 ? void 0 : value[Symbol.asyncIterator]) === 'function';
 }
