diff --git a/src/executors/swc/swc.impl.js b/src/executors/swc/swc.impl.js
index 833487e9c2bb5471779942a90f9f1dd20849edd5..02bf5b2492a15db2dc694abb996ddb3113193e38 100644
--- a/src/executors/swc/swc.impl.js
+++ b/src/executors/swc/swc.impl.js
@@ -80,7 +80,13 @@ function swcExecutor(_options, context) {
             // tmp swcrc with dependencies to exclude
             // - buildable libraries
             // - other libraries that are not dependent on the current project
-            options.swcCliOptions.swcrcPath = (0, inline_2.generateTmpSwcrc)(inlineProjectGraph, options.swcCliOptions.swcrcPath);
+
+            let swcrcPath = options.swcCliOptions.swcrcPath.replace(context.root, '');
+            while (swcrcPath.startsWith('\\') || swcrcPath.startsWith('/')) {
+                swcrcPath = swcrcPath.substring(1);
+            }
+            
+            options.swcCliOptions.swcrcPath = (0, inline_2.generateTmpSwcrc)(inlineProjectGraph, swcrcPath);
         }
         function determineModuleFormatFromSwcrc(absolutePathToSwcrc) {
             var _a, _b;
diff --git a/src/utils/swc/compile-swc.js b/src/utils/swc/compile-swc.js
index 2779e0f499ecb416e4a57a7512b68cf2b0fd7f18..3e5432f5706a0cb8a309aa4d8f70ac00a892ecf8 100644
--- a/src/utils/swc/compile-swc.js
+++ b/src/utils/swc/compile-swc.js
@@ -10,7 +10,7 @@ const print_diagnostics_1 = require("../typescript/print-diagnostics");
 const run_type_check_1 = require("../typescript/run-type-check");
 function getSwcCmd({ swcrcPath, srcPath, destPath }, watch = false) {
     const swcCLI = require.resolve('@swc/cli/bin/swc.js');
-    let swcCmd = `${swcCLI} ${
+    let swcCmd = `npx swc ${
     // TODO(jack): clean this up when we remove inline module support
     // Handle root project
     srcPath === '.' ? 'src' : srcPath} -d ${srcPath === '.' ? `${destPath}/src` : destPath} --config-file=${swcrcPath}`;