diff --git a/polyfills/buffer.js b/polyfills/buffer.js
index 6e5eb931118c7a7ee7fc5b058f5b3f9e91eb7131..4ecc06345357b88aead98b9a3c869cc76b5f5d77 100644
--- a/polyfills/buffer.js
+++ b/polyfills/buffer.js
@@ -1 +1 @@
-export { Buffer } from "buffer";
+export { Buffer } from "node:buffer";
diff --git a/polyfills/process.js b/polyfills/process.js
index 6218fb434cc0b49821d577f8fa0ffa2f909ce596..065ce800e8129176da2fcd5f3740a1d6bbbfb851 100644
--- a/polyfills/process.js
+++ b/polyfills/process.js
@@ -1 +1 @@
-export * as process from "process";
+export * as process from "node:process";
